opennms-base-assembly/src/main/filtered/etc/service-configuration.xml

```diff
@@ -172,15 +172,7 @@ maintained
   </service>
   <service>
     <name>OpenNMS:Name=Provisiond</name>
-    <class-name>org.opennms.netmgt.daemon.SimpleSpringContextJmxServiceDaemon</class-name>
-    <attribute>
-      <name>LoggingPrefix</name>
-      <value type="java.lang.String">provisiond</value>
-    </attribute>
-    <attribute>
-      <name>SpringContext</name>
-      <value type="java.lang.String">provisiondContext</value>
-    </attribute>
+    <class-name>org.opennms.netmgt.provision.service.jmx.Provisiond</class-name>
```

Comment:
@j-white these stats are valuable enough, IMO, to *consider* a rare exception to the intra-*foundation* .rpmnew policy. Can we solve it with an upgrader task? Is this change even a hard requirement? (I'm admittedly foggy on the reasons for the different wiring styles, so apologies if that latter question is objectively dumb)